src/sentry/static/sentry/app/views/organizationIntegrations/integrationIcon.tsx

```diff
@@ -12,16 +12,40 @@ type Props = {
 
 type IconProps = Pick<Props, 'size'>;
 
-const Icon = styled('img')<IconProps>`
+const StyledIcon = styled('img')<IconProps>`
   height: ${p => p.size}px;
   width: ${p => p.size}px;
   border-radius: 2px;
   display: block;
 `;
 
+class Icon extends React.Component<Props> {
+  state = {
+    imgSrc: '',
```

Comment:
@NisanthanNanthakumar I think it would make more sense to use the `DEFAULT_ICON` as the default value here

Reply:
Does it matter, b/c it's going to change in componentDIdMount